2024-08-08 17:13:39,934 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-08-08 17:13:39,934 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /home/ghost/.local/lib/python3.10/site-packages/instabot/bot/bot.py
2024-08-08 17:13:39,935 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-08-08 17:13:40,353 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2024-08-08 17:13:40,765 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2024-08-08 17:13:41,270 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2024-08-08 17:13:41,790 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2024-08-08 17:13:43,016 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2024-08-08 17:13:43,016 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"The password you entered is incorrect. Please try again.","status":"fail","error_title":"Incorrect password","buttons":[{"title":"OK","action":"dismiss"}],"invalid_credentials":true,"exception_name":"UserInvalidCredentials","error_type":"bad_password"}'
2024-08-08 17:13:43,017 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2024-08-08 17:13:43,017 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: The password you entered is incorrect. Please try again.
2024-08-08 17:13:43,017 - instabot version: 0.117.0 (api) - INFO - Error type: bad_password
2024-08-08 17:13:43,017 - instabot version: 0.117.0 (api) - ERROR - Failed to login go to instagram and change your password
2024-08-08 17:13:43,018 - instabot version: 0.117.0 (api) - INFO - Username or password is incorrect.
